docs/source/quick-start/algorithms.rst

```diff
@@ -1,4 +1,254 @@
-Algorithms API Quick Start
---------------------------
+.. include:: common.rst
 
-**UNDER CONSTRUCTION**
+Algorithms API
+--------------
+
+    See the `Algorithms Guide <../guide/algorithms.html>`_ for an overview of
+    this API.
+
+Here is the complete program we are going to use as an example. In sections
+below, we'll break it down into parts and explain what is happening (without
+some of the plumbing details).
+
+.. literalinclude:: ../../examples/algo/complete-example.py
+
+
+Encoding Data
+^^^^^^^^^^^^^
+
+For this quick start, we'll be using the same raw input data file described
+`here <example-data.html>`_ in detail and used for the
+`OPF Quick Start <opf.html>`_. But we will be ignoring the file format and just
+looping over the CSV, encoding one row at a time programmatically before sending
+encodings to the Spatial Pooler and Temporal Memory algorithms.
+
+One Row Of Data
++++++++++++++++
+
+Each row of data in this input file is formatted like this:
+
+::
+
+    7/2/10 9:00,41.5
+
+We need to encode this into three encodings:
+
+* time of day
+* weekend or not
+* scalar value for energy consumption
+
+Creating Encoders
++++++++++++++++++
+
+First, we'll need to create the encoders we'll use to do this:
```

Comment:
I'm having a little difficulty parsing this statement.  Should it read: "First, we'll need to create the encoders we'll use.  To do this:"?